index.js

```diff
@@ -209,6 +213,60 @@ app.get("/popup", function (req, res) {
   res.status(200).send(popupMsg);
 });
 
+const modifyCoordinates = originalResponse =>
+  originalResponse.reduce((acc, curr) => {
+    const {
+      coordinates: {
+        coordinates: [longitude, latitude]
+      }
+    } = curr;
+
+    return [
+      ...acc,
+      {
+        ...curr,
+        coordinates: {
+          latitude,
+          longitude
+        }
+      }
+    ];
+  }, []);
+
+app.use(
+  "/food",
+  createProxyMiddleware({
+    target: config.get("foodUrl"),
+    changeOrigin: true,
+    pathRewrite: { "^/food": "" },
+    selfHandleResponse: true,
+    onProxyRes: responseInterceptor(async responseBuffer => {
+      const originalResponse = JSON.parse(responseBuffer.toString());
+
+      const modifiedResponse = modifyCoordinates(originalResponse);
+
+      return JSON.stringify(modifiedResponse);
+    })
+  })
```

Comment:
Make this re-usable as a new middleware function.

Reply:
ok